### PR TITLE
My first try at proposing hat ideas

### PR DESCRIPTION
**Pull request by gounthar**

### CODE REVIEW

**Review by Thore-Krug — CHANGES_REQUESTED** [review]
You forgot to add the ` .md ` file ending for this File. 

Signed off by: Thore Krug opensource[at]thorekrug.de

**Review by Thore-Krug — COMMENTED** [review]
Is the use of a RTC IC with a External Crystal OK or would a Internal temperature compensated Crystal be prefered ? (  No ICs with a External crystal are available in through hole variants )

File: orange-pi/zero-lts/Seedlings.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+What about getting a hat that hosts
+
+ - an ADC to get soil humidity and temperature via sensors
+ - a 12V 2*fan controller
+ - something to control a solenoid valve (so that we can water the soil)
```
Comment (Thore-Krug) [inline orange-pi/zero-lts/Seedlings.md:5]: What type of solenoid valve ? ( Current, Voltage ? )

Comment (gounthar) [reply orange-pi/zero-lts/Seedlings.md:5]: I was thinking of a 12V solenoid valve like this [one](https://fr.aliexpress.com/item/4001067291113.html?src=google&albch=shopping&acnt=248-630-5778&isdl=y&slnk=&plac=&mtctp=&albbt=Gploogle_7_shopping&aff_atform=google&aff_short_key=UneMJZVf&gclsrc=aw.ds&&albagn=888888&&ds_e_adid=438856512841&ds_e_matchtype=&ds_e_device=c&ds_e_network=u&ds_e_product_group_id=743612850874&ds_e_product_id=fr4001067291113&ds_e_product_merchant_id=107720551&ds_e_product_country=FR&ds_e_product_language=fr&ds_e_product_channel=online&ds_e_product_store_id=&ds_url_v=2&ds_dest_url=https://fr.aliexpress.com/item/4001067291113.html?&albcp=10191220517&albag=107473525088&gclid=CjwKCAiA25v_BRBNEiwAZb4-ZbGWQqmsdY79sAHB94VvdjzUD9ZdLTCrM-g--REJLBNMfptMF9-5oRoC0RwQAvD_BwE)

**Comment by gounthar** [comment]
> You forgot to add the `.md` file ending for this File.
Solved.

**Comment by gounthar** [comment]
> Is the use of a RTC IC with a External Crystal OK or would a Internal temperature compensated Crystal be prefered ? ( No ICs with a External crystal are available in through hole variants )
I gave you the answer in the modified file. Thanks.

**Comment by Thore-Krug** [comment]
Merged. Thank you for your Changes. Merry Christmas and Happy Holiday. @gounthar 

Signed off by Thore Krug opensource[at]thorekrug.de

**Comment by gounthar** [comment]
Thanks a bunch for accepting this PR, U hope you'll enjoy the rest of your holiday @Thore-Krug 😊

**Pull request merged**